### PR TITLE
fix: Override snowflake warehouse

### PR DESCRIPTION
**Pull request by rephus**
Closes:  https://github.com/lightdash/lightdash/issues/10714

### Description:

Before:
Screenshot from 2024-07-23 14-23-43

After:
Screenshot from 2024-07-23 14-23-26
Screenshot from 2024-07-23 14-08-10

Screenshot from 2024-07-23 14-23-10

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: docs/docs/get-started/setup-lightdash/connect-project.mdx [hunk 1]
```diff
@@ -467,6 +467,9 @@ access to use this warehouse. You can see al list of warehouses available in the
   style={{ display: 'block', margin: '0 auto 20px auto' }}
 />
 
+If `Always use this warehouse` is set to yes, then we'll be using this `warehouse` to all queries, even if this is overrwritten
+by the dbt config using `snowflake­_warehouse`
```
Comment (joaoviana) [inline docs/docs/get-started/setup-lightdash/connect-project.mdx:471]:
```suggestion
:::info

If “Always use this warehouse” is set to yes, this warehouse will be used for all queries, even if the dbt configuration specifies a different warehouse using snowflake_warehouse.

:::
```

File: packages/common/src/types/projects.ts [hunk 2]
```diff
@@ -153,6 +153,7 @@ export type CreateSnowflakeCredentials = {
     accessUrl?: string;
     startOfWeek?: WeekDay | null;
     quotedIdentifiersIgnoreCase?: boolean;
+    override?: string;
```
Comment (joaoviana) [inline packages/common/src/types/projects.ts:156]: we should run a `generate-api`

**Review by joaoviana — CHANGES_REQUESTED** [review]
2 changes requested 💪

File: packages/common/src/types/projects.ts (hunk 2 above)
Comment (rephus) [reply packages/common/src/types/projects.ts:156]: I already did, I believe this endpoint is old and not using tsoa, so it doesn't affect the docs.

Comment (joaoviana) [reply packages/common/src/types/projects.ts:156]: Interesting, I got a change in `routes`

Comment (rephus) [reply packages/common/src/types/projects.ts:156]: ups, you were right, sorry. I don't know what happened, I was probably in a different branch. 

Just pushed

**Review by joaoviana — APPROVED** [review]
💪

**Pull request merged**